### PR TITLE
docs: create an intro section about Nerd Fonts

### PR DESCRIPTION
**Pull request by CBID2**
#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

#### Issue 
Relates to #2161
> [!NOTE] 
This PR fulfills the fifth bullet point of my RSOD proposal.

### CODE REVIEW

File: docs/books/nvchad/nerd_fonts.md [hunk 1]
```diff
@@ -11,11 +11,11 @@ tags:
 
 # :material-format-font: Nerd Fonts - Fonts for Developers
 
-Installation of Nerd Fonts does not change the function of Neovim or NvChad, but simply adds fonts that are more pleasing to the eye than whatever the standard terminal font is currently.
+## :material-format-font: What is Nerd Fonts?
 
-![Nerd Fonts](images/nerd_fonts_site_small.png){ align=right } Nerd Fonts are a collection of modified fonts aimed at developers. In particular, "iconic fonts" such as Font Awesome, Devicons, Octicons etc. are used to add extra glyphs.
+![Nerd Fonts](images/nerd_fonts_site_small.png){ align=right } Nerd Fonts are a collection of modified fonts aimed at developers. In particular, "iconic fonts" such as [Font Awesome](https://fontawesome.com/), [Devicons](https://devicon.dev/), and [Octicons](https://primer.style/foundations/icons) are use to add extra glyphs.
```
Comment (SergeCroise) [inline docs/books/nvchad/nerd_fonts.md:16]:
```suggestion
![Nerd Fonts](images/nerd_fonts_site_small.png){ align=right } Nerd Fonts are a collection of modified fonts aimed at developers. In particular, "iconic fonts" such as [Font Awesome](https://fontawesome.com/), [Devicons](https://devicon.dev/), and [Octicons](https://primer.style/foundations/icons) are used to add extra glyphs.
```

File: docs/books/nvchad/nerd_fonts.md [hunk 2]
```diff
@@ -11,11 +11,11 @@ tags:
 
 # :material-format-font: Nerd Fonts - Fonts for Developers
 
-Installation of Nerd Fonts does not change the function of Neovim or NvChad, but simply adds fonts that are more pleasing to the eye than whatever the standard terminal font is currently.
+## :material-format-font: What is Nerd Fonts?
```
Comment (SergeCroise) [inline docs/books/nvchad/nerd_fonts.md:14]:
```suggestion
## :material-format-font: What are Nerd Fonts?
```

File: docs/books/nvchad/nerd_fonts.md [hunk 3]
```diff
@@ -11,11 +11,11 @@ tags:
...
+![Nerd Fonts](images/nerd_fonts_site_small.png){ align=right } Nerd Fonts are a collection of modified fonts aimed at developers. In particular, "iconic fonts" such as [Font Awesome](https://fontawesome.com/), [Devicons](https://devicon.dev/), and [Octicons](https://primer.style/foundations/icons) are use to add extra glyphs.
 
-Nerd Fonts takes the most popular programming fonts and modifies them by adding a group of glyphs (icons). A font patcher is also available if the font you'd like to use has not already been edited. A convenient preview is available on the site, allowing you to see how the font should look in the editor. For more information, check the project's main [site](https://www.nerdfonts.com/).
+Nerd Fonts also takes the most popular programming fonts such as MonoLisa or SourceCode Pro and modifies them by adding a group of glyphs (icons). A font patcher is available if the font you'd like to use has not already been edited.  There's also a preview feature where you can see how the font should look in the editor. For more information, check out the project's main [site](https://www.nerdfonts.com/).
```
Comment (SergeCroise) [inline docs/books/nvchad/nerd_fonts.md:18]:
```suggestion
Nerd Fonts also takes the most popular programming fonts such as MonoLisa or SourceCode Pro and modifies them by adding a group of glyphs (icons). A font patcher is available if the font you would like to use has not already been edited.  There is also a preview feature where you can see how the font should look in the editor. For more information, check out the project's main [site](https://www.nerdfonts.com/).
```

**Comment by CBID2** [comment]
Made the changes @SergeCroise

**Comment by sspencerwire** [comment]
@CBID2 Thanks for your interest in modifying this document. Unfortunately, (and perhaps this is a problem with your RSOD list in general) In order to qualify for the RSOD, it must be a new document, rather than an edit to an existing document,  and fit within the "Desktop" category [found here](https://docs.rockylinux.org/desktop/). Modifying existing documents, while helpful, does not fulfill the goals of the RSOD. @wsoyinka Please chime in here as well.

**Comment by sspencerwire** [comment]
@all-contributors add @CBID2 for content

**Pull request merged**